### DISCUSSION TITLE
Teampass 3.0.4

### DISCUSSION THREAD

**Discussion by nilsteampassnet** [nilsteampassnet/TeamPass#3651] [General]
## What's new
This release embeds next fixes:

- Fix for #3650
- Full list of changes: [3.0.3 -> 3.0.4](https://github.com/nilsteampassnet/TeamPass/compare/3.0.3...3.0.4?expand=1)

> Please ALWAYS download from [the latest commit](https://github.com/nilsteampassnet/TeamPass/archive/refs/heads/master.zip) as this project is evolving regularly.

## Languages

Please join [Teampass v3 translation project on Poeditor](https://poeditor.com/projects/view?id=433631) and translate it for your language.

## Installation

Follow instructions from [Documentation](https://documentation.teampass.net/#/install/installation).

## Upgrade

Follow instructions from [Documentation](https://documentation.teampass.net/#/install/upgrade).

## Ideas and comments

Are welcome ... please use [Discussions](https://github.com/nilsteampassnet/TeamPass/discussions).

<hr /><em>This discussion was created from the release <a href='https://github.com/nilsteampassnet/TeamPass/releases/tag/3.0.4'>Teampass 3.0.4</a>.</em>

**Comment by abdullah4142** [comment]
Hello, I installed version 3.0.3, but when I try to log in from Active Directory with LDAP, it stays as "In Progress" and does not log in. What should I do ? Could you help ? Thanks in advance.

**Comment by BroenGuiraldson** [comment]
Hello,
When I follow the upgrade instructions from [Documentation](https://documentation.teampass.net/#/install/upgrade), I receive this error message when I launch the `git pull` command to upgrade from v3.0.3 to the latest:
_error: Impossible de tirer avec un rebasage : vous avez des modifications non indexées.
error: veuillez les valider ou les remiser._

I never practice Git so it's disturbing for newbies like me to upgrade with this solution when you never dit it before with other projects.
I will try to follow the manual upgrade by searching how to push it to a non graphic Debian.

**Reply by BroenGuiraldson** [reply]
Looks like `git fetch --all` + `git reset --hard origin/master` permit me to upgrade.
